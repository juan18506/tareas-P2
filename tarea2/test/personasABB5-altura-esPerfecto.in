crearArbolPersonas

# arbol con 0 elementos - altura 0
alturaArbolPersonas

# arbol es perfecto
esPerfectoArbolPersonas

crearAgenda
crearPersona 5 55 Persona5
crearFecha 5/55/55
crearEvento 5 Descripcion5
agregarEventoPersona
insertarPersonaArbolPersonas

# arbol con 1 elemento - altura 1
alturaArbolPersonas

# arbol es perfecto
esPerfectoArbolPersonas

crearAgenda
crearPersona 10 100 Persona10
insertarPersonaArbolPersonas

# arbol con 2 elementos - altura 2
alturaArbolPersonas

# arbol no es perfecto
esPerfectoArbolPersonas

crearAgenda
crearPersona 2 22 Persona2
insertarPersonaArbolPersonas

# arbol con 3 elementos - altura 2
alturaArbolPersonas

# arbol es perfecto
esPerfectoArbolPersonas

crearAgenda
crearPersona 20 200 Persona20
insertarPersonaArbolPersonas

crearAgenda
crearPersona 7 77 Persona7
insertarPersonaArbolPersonas

# subarboles con alturas distintas
# arbol no es perfecto
esPerfectoArbolPersonas

crearAgenda
crearPersona 1 11 Persona1
insertarPersonaArbolPersonas

crearAgenda
crearPersona 3 33 Persona3
insertarPersonaArbolPersonas

# arbol con 7 elementos - altura 3
alturaArbolPersonas

# arbol es perfecto
esPerfectoArbolPersonas

# remuevo la persona 20 - altura 3
removerArbolPersonas 20
alturaArbolPersonas

# arbol no es perfecto
esPerfectoArbolPersonas

# remuevo la persona 5
removerArbolPersonas 5

# remuevo la persona 2
removerArbolPersonas 2

crearAgenda
crearPersona 4 44 Persona4
# inserto la persona 4
insertarPersonaArbolPersonas

alturaArbolPersonas
esPerfectoArbolPersonas

liberarArbolPersonas

Fin
