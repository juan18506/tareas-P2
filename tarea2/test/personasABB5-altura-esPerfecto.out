1>2>#  arbol con 0 elementos - altura 0.
3>La altura del árbol de personas es: 0.
4>#  arbol es perfecto.
5>El árbol de personas es perfecto.
6>7>8>9>10>11>12>#  arbol con 1 elemento - altura 1.
13>La altura del árbol de personas es: 1.
14>#  arbol es perfecto.
15>El árbol de personas es perfecto.
16>17>18>19>#  arbol con 2 elementos - altura 2.
20>La altura del árbol de personas es: 2.
21>#  arbol no es perfecto.
22>El árbol de personas NO es perfecto.
23>24>25>26>#  arbol con 3 elementos - altura 2.
27>La altura del árbol de personas es: 2.
28>#  arbol es perfecto.
29>El árbol de personas es perfecto.
30>31>32>33>34>35>36>#  subarboles con alturas distintas.
37>#  arbol no es perfecto.
38>El árbol de personas NO es perfecto.
39>40>41>42>43>44>45>#  arbol con 7 elementos - altura 3.
46>La altura del árbol de personas es: 3.
47>#  arbol es perfecto.
48>El árbol de personas es perfecto.
49>#  remuevo la persona 20 - altura 3.
50>51>La altura del árbol de personas es: 3.
52>#  arbol no es perfecto.
53>El árbol de personas NO es perfecto.
54>#  remuevo la persona 5.
55>56>#  remuevo la persona 2.
57>58>59>60>#  inserto la persona 4.
61>62>La altura del árbol de personas es: 4.
63>El árbol de personas NO es perfecto.
64>65>Fin.
